lib/DXIL/DxilShaderModel.cpp

```diff
@@ -79,114 +79,111 @@ const ShaderModel *ShaderModel::Get(Kind Kind, unsigned Major, unsigned Minor) {
   /* <py::lines('VALRULE-TEXT')>hctdb_instrhelp.get_shader_model_get()</py>*/
   // VALRULE-TEXT:BEGIN
   const static std::pair<unsigned, unsigned> hashToIdxMap[] = {
-      {1024, 0},    // ps_4_0
```

Comment:
A lot of the diffs here are because the generated code wasn't run through clang-format, so this should clean up and dramatically simplify the patch once it is formatted correctly.

Reply:
Earlier I was able to apply codeformatting through github-actions and it used the right clang-format version, didn't work this time I guess due to the diff size. Anyway did it locally and pushed.

Reply:
Resolved